src/middlewares/multipartFormDataParser.js

```diff
@@ -0,0 +1,49 @@
+const BusBoy = require('busboy')
+const contentTypeLib = require('content-type')
+const createError = require('http-errors')
+
+module.exports = () => ({
+  before: async (handler, next) => {
+    const { headers } = handler.event
+    if (!headers) {
+      return next()
+    }
+
+    const contentType = headers['content-type']
+    if (contentType) {
```

Comment:
Missed this one before...

We should something like below:

```javascript
if (typeof contentType === 'undefined') {
  return next()
}
```

Reply:
I implemented this as a simple else clause, any particular reason to check the type? Seems we just need to know if contentType is false or not.

Reply:
Not really. Kind of a default practice to be honest. In this case, it might actually make more sense to do a *falsy* check as we want to check if the content type is `null` or `false`...